### ISSUE TITLE
Only add session/streams props to child elements that accept them

### ISSUE THREAD

**Issue by aiham** [aiham/opentok-react#16]
If you add an element such as a \<div\> inside OTSession or OTStreams you get this warning:

```
Warning: Unknown props `session`, `streams` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```

These props should only be added if the child element lists them in their propTypes.

**Comment by sstawecki** [comment]
Hi @aiham, what do you think about using react context to solve this issue? or this library [react-broadcast](https://github.com/mjackson/react-broadcast) which uses a context-solution internally.

**Comment by aiham** [comment]
@sstawecki Thanks for the suggestion! I'll look into it.

**Comment by aiham** [comment]
Fixed in #24. Thank you @emguide!
